supplementary_style_guide/style_guidelines/general-formatting.adoc

```diff
@@ -16,27 +16,54 @@ When the format _day Month year_ causes a presentation or clarity issue, use _Mo
 The Conscious Language Group supports the Red Hat commitment to remove problematic language from our code, documentation, websites, and open source projects with which Red Hat is involved.
 For more information about the Conscious Language Group, see https://github.com/conscious-lang/conscious-lang-docs.
 
+
 === blacklist and whitelist
 
-When possible, rewrite documentation to avoid these terms.
+When it is not possible to remove the terms _blacklist_ and _whitelist_, replace them with one of the following alternatives:
 
+* blocklist / allowlist: This combination is recommended by _The IBM Style Guide_. Use this combination unless your product area has another specific replacement that is agreed between engineering leadership and your documentation team.
+* denylist / allowlist
+* blocklist / passlist
 
-.Example: Removing 'blacklist'
-image:images/no.png[no] Heat blacklists any servers in the list from receiving updated heat deployments. After the stack operation completes, any blacklisted servers remain unchanged. You can also power off or stop the `os-collect-config` agents during the operation.
+.Examples
+* Removing blacklist
 
-image:images/yes.png[yes] Heat excludes any servers in the list from receiving updated heat deployments. After the stack operation completes, any excluded servers remain unchanged. You can also power off or stop the `os-collect-config` agents during the operation.
+image:images/no.png[no] Heat _blacklists_ any servers in the list from receiving updated heat deployments. After the stack operation completes, any blacklisted servers remain unchanged. You can also power off or stop the `os-collect-config` agents during the operation.
 
+image:images/yes.png[yes] Heat _excludes_ any servers in the list from receiving updated heat deployments. After the stack operation completes, any excluded servers remain unchanged. You can also power off or stop the `os-collect-config` agents during the operation.
 
-.Example: Removing 'whitelist'
-image:images/no.png[no] The following steps demonstrate adding a new rule to whitelist a custom binary.
+* Removing whitelist
 
-image:images/yes.png[yes] The following steps demonstrate adding a new rule to allow a custom binary.
+image:images/no.png[no] The following steps demonstrate adding a new rule to _whitelist_ a custom binary.
+
+image:images/yes.png[yes] The following steps demonstrate adding a new rule to _allow_ a custom binary.
+
+
+=== master and slave
+
+You can use the following alternatives for _master_ /_slave_:
```

Comment:
This list seems restrictive. What about parent/child? Maybe the introductory statement to the list should mention that the following are examples of alternative terms. WDYT?

I think a note stating that one should consult with their project team to agree on a selection of alternative terms that would suit the upstream and downstream community of a certain product would not go amiss. I have had engineers and QE express concern that alternative terms were selected without their input.

Reply:
The conscious language group have offered these options as ".. teams should follow the recommendations outlined by the Linux kernel. These recommendations have informed the terminology recommendations we provide below.

- primary / secondary
- source / replica
- initiator, requester / responder
- controller, host / device, worker, proxy
- director / performer
"
I agree about the consulting with engineering teams. In the previous entry about black/white list, I added this line "Use this combination unless your product area has another specific replacement that is agreed between engineering leadership and your documentation team." -so we could include something like that more explicitly, I guess at the top of this section wold make sense.

The conscious language group are clear that we need to consult with our product teams, so you could escalate the query up to product management, who might have already fixed on replacement terms but it wasn't communicated as far as docs yet.

This is a bit from the **Inclusive Language Project: FAQ** on the Source:
"To ensure consistency and success, it is imperative for product team stakeholders to align internally. For example, documentation teams should engage in discussions with their engineering leadership to reach an agreement on replacement terms. This will ensure that the product documentation will match the code.

If your project opts to use some other term or phrase (for example, Fedora chose ‘rawhide’ as their replacement for their master branch) please let us know that you’ve done so, and why, so that we can learn from your decisions."

Reply:
Thanks, @ndeevy . Finding information on the Source can be difficult when people need to get docs out. I think a note at the start of this section would help that explains that a writer should seek stakeholder agreement when selecting a suitable alternative term for a specific product. I cannot see the start of this section on the PR, so maybe such a note already exists before the description of each term to replace.

Reply:
@dfitzmau @ndeevy There's a link to the Github repo for the Conscious Language Program in the PR's first paragraph. Jump to there and the same info appears.

Reply:
Thanks, @stoobie . I expanded the PR and found that link. I still had to go digging to find a statement about seeking agreement from project stakeholders. I still think having a short note at the start of the section would help writers avoid getting egg on their face when stakeholders raise concerns about alternative term choice. Getting stakeholder buy-in before adding a term to a product's docs would really alleviate any concerns on a product team.

Reply:
@dfitzmau Understood. I just wanted everyone to be aware that in this case, we don't have to depend on the Source being the source of truth, that there's a public site with the info. I think saying something about stakeholder collaboaration/managment is fine.

Reply:
Hi @Amrita42 . Might I ask the decision on my query?

The statement on the GitHub source is perfect, but it's buried and does not stand out among the README text. 

Can we add this statement (that follows) as an important notice to the CCS guide? I personally experienced resistance from engineering when I replaced an undesired legacy term with one of the alternatives from the list on this PR.

_To ensure consistency and success, it is imperative for product team stakeholders to align internally. For example, documentation teams should engage in discussions with their engineering leadership to reach an agreement on replacement terms. This will ensure that the product documentation will match the code._

Reply:
@dfitzmau  imo I  think we can add a statement at the start of the section. Will add that.

Reply:
Modified the sentence to reflect present tense